packages/frontend/component/src/ui/modal/modal.stories.tsx

```diff
@@ -139,3 +139,37 @@ export const Animations = () => {
     </div>
   );
 };
+
+export const Nested = () => {
+  const [openRoot, setOpenRoot] = useState(false);
+  const [openNested, setOpenNested] = useState(false);
```

Comment:
we definitely need `[visible, open, close] = useToggle(false)`

Reply:
If so, `toggle()` is also necessary 🥲